test/.eslintrc.json

```diff
@@ -0,0 +1 @@
+{"env":{"mocha":true}, "rules": { "no-shadow": ["error", { "allow": ["done"] }] }}
```

Comment:
I think we should format this, like the eslintrc in the root dir, as YAML. Just for consistency. 
Having it as a one-line JSON is somewhat nice, but I think I prefer the readability of YAML for these.